source/includes/_collectionOfWishlistPremium.md

```diff
@@ -2,6 +2,36 @@
 
 The Swym platform supports having multiple wishlists for a user, eg: my lounge, my bedroom, etc. or in case of fashion eg: my outdoor collection, my summer list, etc. These APIs are available on the premium plans, please contact us to enable these APIs for your installation.
 
+
+## Render the Add to Wishlist Collections Pop up.
+
+### _swat.addToWishList(SwymPageData, callbackFn, false, variantInfo, {edit: true, evt: e});
+
+```javascript 
+window._swat.addToWishList(SwymPageData, function successCallBackFn(e) {
```

Comment:
Also we should have SwymPageData.et as 4 right? We need to mention it wherever we change et from SwymPageData.

Reply:
Instead of `e`, use `response` or `res` or `r` in callback function.

Reply:
Also, have you done POC for this same params mentioned here? Check and confirm if actual et 4 is done and shows up on fetch call i.e. shows in notepad.